### ISSUE TITLE
systemd 228 adding services to /storage/.config/system.d

### ISSUE THREAD

**Issue by MilhouseVH** [OpenELEC/OpenELEC.tv#4470]
When booting with systemd-228 (#4458), a whole bunch of new services are being added to `/storage/.config/systemd.d` which results in slower startup and log messages appearing at startup.

Prior to systemd-228, this is the contents of a stock `/storage/.config/system.d`, eg. [build #1119](http://forum.kodi.tv/showthread.php?tid=238393&pid=2164900#pid2164900):
```
OpenELEC (unofficial) Version: devel-20151119035205-#1119-g785752e
OpenELEC git: Commit:785752e
OpenELEC:~ # ls -la /storage/.config/system.d
total 40
    32 -rw-r--r--    1 root     root         28816 Nov 28 04:03 README
     4 -rw-r--r--    1 root     root          1772 Nov 28 04:03 cifs.mount.sample
     4 -rw-r--r--    1 root     root          1729 Nov 28 04:03 nfs.mount.sample
```
Once OpenELEC with systemd-228 is booted, eg. [build #1121](http://forum.kodi.tv/showthread.php?tid=238393&pid=2167501#pid2167501), `/storage/.config/system.d` becomes:
```
OpenELEC (unofficial) Version: devel-20151121220629-#1121-g297af74
OpenELEC git: Commit:297af74
OpenELEC:~ # ls -la /storage/.config/system.d
total 108
drwxr-xr-x   17 root     root          4096 Nov 28 04:07 .
drwxr-xr-x   17 root     root          4096 Nov 28 04:03 ..
-rw-r--r--    1 root     root         28816 Nov 28 04:03 README
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 basic.target.wants
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 bluetooth.target.wants
-rw-r--r--    1 root     root          1772 Nov 28 04:03 cifs.mount.sample
lrwxrwxrwx    1 root     root            37 Nov 28 04:07 ctrl-alt-del.target -> /usr/lib/systemd/system/reboot.target
lrwxrwxrwx    1 root     root            46 Nov 28 04:07 dbus-fi.epitest.hostap.WPASupplicant.service -> /usr/lib/systemd/system/wpa_supplicant.service
lrwxrwxrwx    1 root     root            35 Nov 28 04:07 default.target -> /usr/lib/systemd/system/kodi.target
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 graphical.target.wants
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 halt.target.wants
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 installer.target.wants
lrwxrwxrwx    1 root     root            37 Nov 28 04:07 kbrequest.target -> /usr/lib/systemd/system/rescue.target
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 kodi.service.wants
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 kodi.target.wants
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 local-fs.target.wants
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 multi-user.target.wants
-rw-r--r--    1 root     root          1729 Nov 28 04:03 nfs.mount.sample
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 poweroff.target.wants
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 reboot.target.wants
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 sleep.target.wants
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 sockets.target.wants
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 sysinit.target.wants
drwxr-xr-x    2 root     root          4096 Nov 28 04:07 textmode.target.wants
```

With these extra services the following message now appears at startup (I'm on a wired connection):

![screenshot](http://i.imgur.com/vAawcMt.jpg)

If you now roll back to any previous version of OpenELEC, even one before systemd-228, you will still have these extra services executing at startup that are created by systemd-228, so you continue to have the "waiting..." message at startup after rolling back to [build #1119](http://forum.kodi.tv/showthread.php?tid=238393&pid=2164900#pid2164900), or any other build.

I'm assuming this is not intentional. And I also wonder if it may be responsible for the WiFi related issues (ie. not connecting automatically) that started [being reported](http://forum.kodi.tv/showthread.php?tid=238393&pid=2167796#pid2167796) with my test builds after the move to systemd-228, and continued even when using a [non-system-228 build](http://forum.kodi.tv/showthread.php?tid=238393&pid=2168521#pid2168521).

**Comment by stefansaraev** [comment]
works fine for me. 
are you sure you have https://github.com/OpenELEC/OpenELEC.tv/blob/master/packages/sysutils/systemd/patches/systemd-06_stop-unit_file_preset_all-doing-stupid-things.patch in your builds ?

**Comment by MilhouseVH** [comment]
Yes, my builds have that patch.

**Comment by MilhouseVH** [comment]
After discussing with @stefansaraev on #irc, this will be fixed by #4472 - the current `systemd-06_stop-unit_file_preset_all-doing-stupid-things.patch` is no longer compatible with systemd-228.
